python/res/fm/ecl/ecl_run.py

```diff
@@ -129,10 +129,11 @@ def numCpu(self):
 
 
     def initEnv(self):
```

Comment:
Should this function get a new name and become private?

Reply:
I missed this comment. Do you have a suggestion for a new name? Do you mean prefix with _? Just for clarification, what does this "private" actually signify? That it is unsafe to call? That it has side effects (which this one does not)? That we hope no one calls it? Or that we have checked that no one is calling it?

Reply:
It is mere social convention. If you are interaction with a `_` prefixed function or field in Python that is not a local variable or from `self` you are breaking the social contract between Python programmers and all your thoughts about semver, correctness guaranteed by an API provider etc is out the window.

Reply:
How about `_build_simulator_environment(self)`?

Reply:
Ok, so by extension, all functions not prefixed with _ come with a guarantee that we won't change the behavior? If so, we shouldn't rename or make it private or change what it does at all maybe?

Reply:
After discussion with @sondreso I conclude that renaming it as you suggest makes sense. _ means that I do not intend anyone outside this module to call it. Lack of _ means that it is intended to be called by other modules in libres. (for some definition of module). No guarantees are attached to either name convention, unless otherwise specified. Lack of _ probably should not mean that the symbol is intended for consumers of libres in general, only internally in the library.